app/assets/stylesheets/components/_index-table.scss

```diff
@@ -51,36 +53,84 @@ It expects to be the primary content of the given page (full width).
     &:first-child {
       padding-left: 3rem;
     }
+
     &:last-child {
       padding-right: 3rem;
     }
   }
 
   &__row-header {
+    position: sticky;
+    left: 0;
+    z-index: 1;
+    background-color: inherit;
     white-space: nowrap;
-    padding: 0.8rem $s25/2;
+    padding: 0.8rem $s25 / 2;
+  }
+
+  &__name-column {
```

Comment:
I'm wondering whether it makes sense to call this &__name-column, or if we could call it something a bit more generic like &__sticky-column so that using this class makes sense regardless of what type of index table you're using it on. I guess the only other example where we're doing something similar is the metrics dashboard table and that's also a name, albeit an organization name..